### PR TITLE
Intents handling

### LINKED ISSUE #28: Handling one-time photo/video for other apps
I don't even know where to start implementing that, but camera apps installed as system app should be able to handle other apps requests for a single photo/video (e.g. when uploading a photo from Chrome)

This can be done later once we start testing the app inline with LineageOS

### PR DESCRIPTION
**Pull request by SebaUbuntu**

### CODE REVIEW

**Comment by SebaUbuntu** [comment]
Right now the fragments are drawn on top of main activity but stuff below is still clickable

It might be worth to move the entire main activity to a fragment and swapping taking <-> preview with fragment transactions, this would solve the issue

**Comment by SebaUbuntu** [comment]
Plus we need to use activity-alias

**Comment by luk1337** [comment]
Fix lint errors pls ^^

File: app/src/main/java/org/lineageos/selfie/ui/VideoPreviewFragment.kt [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import androidx.fragment.app.Fragment
+import org.lineageos.selfie.R
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [VideoPreviewFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class VideoPreviewFragment : Fragment() {
+    private var uri: Uri? = null
```
Comment (luk1337) [inline app/src/main/java/org/lineageos/selfie/ui/VideoPreviewFragment.kt]: lateinit var?

**Comment by SebaUbuntu** [comment]
While sending photo result intent UI freezes for a few seconds, with video it doesn't do that because it passes a URI, it's probably because the bitmap size is too big, do you know if we need to reduce the resolution and/or compress the byte array?

**Pull request closed without merging**